### ISSUE TITLE
date and year fields in m_reference

### ISSUE THREAD

**Issue by mdlux** [NINAnor/NOFA#6]
Are both fields required?

**Comment by andersfi** [comment]
I think the usage would be:
Year: for ordinary references, like reports, papers, books etc..
Date: for e.g. interview surveys, personal correspondence or similar. 

both would obviously not be used at the same time.

**Comment by andersfi** [comment]
revised m_refference in temporary schema for review and suggestions.

**Comment by mdlux** [comment]
I see a table named:
l_refferences_new
Is it what you mean?

**Comment by andersfi** [comment]
Yes, it's shortened considerably by removing unused references.



Sendt fra min Samsung-enhet


-------- Opprinnelig melding --------
Fra: Matteo De Stefano <notifications@github.com>
Dato: 28.03.2017 15:50 (GMT+01:00)
Til: NINAnor/NOFA <NOFA@noreply.github.com>
Ko: "Anders G. Finstad" <anders.finstad@ntnu.no>, Comment <comment@noreply.github.com>
Emne: Re: [NINAnor/NOFA] date and year fields in m_reference (#6)


I see a table named:
l_refferences_new
Is it what you mean?

-

**Comment by ninsbl** [comment]
> I think the usage would be:
> Year: for ordinary references, like reports, papers, books etc..
> Date: for e.g. interview surveys, personal correspondence or similar.
> 
> both would obviously not be used at the same time.

Exactly! I would say this is best handled in a validation / signal and slot in the add or modify reference in the Plugin GUI...
@mdlux  Would you agree on that (and in case close the issue)?
